evalml/automl/automl_search.py

```diff
@@ -591,9 +592,10 @@ def _add_baseline_pipelines(self, X, y):
             baseline = ModeBaselineBinaryPipeline(parameters={})
         elif self.problem_type == ProblemTypes.MULTICLASS:
             baseline = ModeBaselineMulticlassPipeline(parameters={})
-        else:
+        elif self.problem_type == ProblemTypes.REGRESSION:
             baseline = MeanBaselineRegressionPipeline(parameters={})
-
+        else:
+            baseline = TimeSeriesBaselineRegressionPipeline(parameters={"pipeline": {"gap": 0, "max_delay": 0}})
```

Comment:
I think we should use the problem configuration parameters passed by the user to ensure the shifting between targets and predictions is accurate.

Reply:
right, thanks!